docs/docs/dev_docs/getting_started/sandbox.md

```diff
@@ -247,8 +250,8 @@ Running now should yield output:
     rollupAddress: EthAddress {
       buffer: <Buffer cf 7e d3 ac ca 5a 46 7e 9e 70 4c 70 3e 8d 87 f6 34 fb 0f c9>
     },
-    client: 'pxe@0.1.0',
-    compatibleNargoVersion: '0.11.1-aztec.0'
+    client: 'pxe@#include_aztec_short_version'',
+    compatibleNargoVersion: '#include_noir_version'
```

Comment:
Should some of the underlying print not be altered as well? From the `initial 1000000` wording seems to be the old one.